MathMagician/Numbers/Integer.cs

```diff
@@ -9,8 +9,8 @@ namespace MathMagician.Numbers
     interface Integer
     {
         int GetFirst();
-        int GetNext(int current);
-        int GetSequence(int how_many);
-        string printNumbers(int[] how_many);
+        int GetNext();
```

Comment:
@utleroy Why did you make this change? This causes your `GetNext` [implementation](https://github.com/nss-evening-cohort-04/mathemagician/pull/16/files#diff-806efad2d459e2d24bae6507563a1908R29) and [test](https://github.com/nss-evening-cohort-04/mathemagician/pull/16/files#diff-1c5e2ed523e4380e3dfac6e6a681e3b6R40) to not make sense.